db/migrate/20210906101108_create_delayed_jobs.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+class CreateDelayedJobs < ActiveRecord::Migration[6.1]
+  def self.up
+    create_table :delayed_jobs do |table|
+      # Allows some jobs to jump to the front of the queue
+      table.integer :priority, default: 0, null: false
+      # Provides for retries, but still fail eventually.
+      table.integer :attempts, default: 0, null: false
+      # YAML-encoded string of the object that will do work
+      table.text :handler,                 null: false
+      # reason for last failure (See Note below)
+      table.text :last_error
+      # When to run. Could be Time.zone.now for immediately, or sometime in the future.
+      table.datetime :run_at
+      # Set when a client is working on this object
+      table.datetime :locked_at
+      # Set when all retries have failed (actually, by default, the record is deleted instead)
+      table.datetime :failed_at
+      # Who is working on this object (if locked)
+      table.string :locked_by
+      # The name of the queue this job is in
+      table.string :queue
+      table.timestamps null: true
+    end
```

Comment:
At least, we don't have another dependency on e.g. redis.

Reply:
Yes, that is exactly the reason we chose delayed_job.